gnd/src/test/java/com/google/android/gnd/persistence/local/LocalDataStoreTest.java

```diff
@@ -167,137 +207,97 @@ public void setUp() {
 
   @Test
   public void testGetProjects() {
-    Project project1 =
-        Project.newBuilder()
-            .setId("id 1")
-            .setTitle("project 1")
-            .setDescription("foo description")
-            .build();
-    Project project2 =
-        Project.newBuilder()
-            .setId("id 2")
-            .setTitle("project 2")
-            .setDescription("foo description 2")
-            .build();
-    localDataStore.insertOrUpdateProject(project1).test().assertComplete();
-    localDataStore.insertOrUpdateProject(project2).test().assertComplete();
-    localDataStore
-        .getProjects()
-        .test()
-        .assertValue(ImmutableList.<Project>builder().add(project1, project2).build());
+    Project project = TEST_PROJECT;
```

Comment:
Please use constants directly, here and throughout; extra indirection makes tests [marginally] harder to ready.